### ISSUE TITLE
Notifications are executed on all nodes

### ISSUE THREAD

**Issue by faelau** [leg100/otf#709]
Hi,

we are running a three node OTF instance. It's all running fine, except for sending notifications. The notifications are sending successfully, but are triggered three times (equally to the count of nodes).
I think they are supposed to be send only once for each run?

**Comment by leg100** [comment]
Yup only once. Something is going awry. I'll take a look.

**Comment by faelau** [comment]
Hi, I'm getting a migration error if upgrading to `0.3.6`:

```
Error: creating database pool: migrating database: 002_rename_agents_to_runners.sql: ERROR: function public.runners_notify_event() does not exist (SQLSTATE 42883)
```

**Comment by faelau** [comment]
```
otf=> \dn
        List of schemas
   Name    |       Owner       
-----------+-------------------
 otf       | otf
 pgbouncer | postgres
 public    | pg_database_owner
```

I have indeed multiple schemas in the postgres database. That's due to [PostgreSQL 15 release](https://www.postgresql.org/docs/release/15.0/) and the deprecation of `CREATE` on the schema `public`.

**Comment by leg100** [comment]
Fixed in e7510b0fabb487f76a6f5122bb6e2708fadb3a75

@faelau if you still have issues, please re-open.

**Comment by faelau** [comment]
I've indeed still issues. After updating, even planning a run fails (or more exactly running into a timeout). In the log the following is printed:

```
2025/01/21 01:00:57 ERROR retrieving type for database event component=broker error="resource not found" table=jobs id=run-LPKE2aU89Ko3hxoh/plan action=INSERT
```

@leg100 Don't have the permission to re-open!
